### ISSUE TITLE
Accessibility QA of site

### ISSUE THREAD

**Issue by JessicaMarine1** [GSA/smartpay-website#233]
Before the new GSA SmartPay program site goes live, we need to do some manual & automated testing, to ensure the new program site is accessible:

- [x] Run lighthouse
- [x]  508 testing
   - [x]  [AMP](https://insite.gsa.gov/employee-resources/information-technology/it-accessibility-section-508/accessibility-solutions/accessibility-management-platform-amp?check_logged_in=1)
   - [x]  [WAVE evaluation tool](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
   - [x]  Manual testing (see pic below)
- [x]  Log any issues discovered

**Comment by JessicaMarine1** [comment]
![Image](https://github.com/GSA/smartpay-website/assets/92383362/874d04ed-30f4-416d-9e37-1c3dc43a0c73)

**Comment by JessicaMarine1** [comment]
@brentryanjohnson Is this something that you'd be willing to work on?

**Comment by brentryanjohnson** [comment]
I added Lighthouse metrics to the 🔒 [ IA spreadsheet](https://docs.google.com/spreadsheets/d/1-nIXpGYsCgKcrXA4LLTEECBILCZOVWmn4n4abvFJGZU/edit?usp=sharing).

Some notes:

- I suspect cloud.gov Pages throttles performance for previews, so `Performance` metrics might be off (compared to production)
- cloud.gov Pages previews intentionally block search indexing (so that previews don't show up in search results), so SEO results reflect that.
  - That said, the website lacks `meta` descriptions across the board, noted in #250 (I think @mark-meyer is working on this)
- I didn't run Lighthouse on all 56 tax pages, but I took a sample of 5 pages (and ran Arizona, one of the most complicated, through both pa11y and Lighthouse), so the results should be largely true for all state tax pages, given they use the same template.
- I noted only one accessibility error among all pages, captured in #255
